test/test-v8debugapi.js

```diff
@@ -80,6 +80,16 @@ describe('v8debugapi', function() {
       });
     });
 
+  it('should disambiguate incorrect path if filename is unique',
+    function(done) {
+      require('./fixtures/foo.js');
+      var bp = { id: 0, location: {line: 1, path: '/test/foo.js'}};
+      api.set(bp, function(err) {
+        assert.ifError(err);
+        api.clear(bp);
+        done();
+      });
+    });
```

Comment:
Test added using `hello.js` which exists in the fixtures root as well as the `a` and `b` subdirectories. We can successfully set a breakpoint in `/Server/a/hello.js` despite `/Server/` being incorrect and not part of the actual path.